src/routing/pages/probe/Explore.svelte

```diff
@@ -10,13 +10,12 @@
 
   function handleBodySelectors(event) {
     const { selection, type } = event.detail;
-    if (type === "percentiles") store.setField("visiblePercentiles", selection);
-    if (type === "timeHorizon") store.setField("timeHorizon", selection);
-    if (type === "metricType")
-      store.setField("proportionMetricType", selection);
-    if (type === "activeBuckets") {
-      store.setField("activeBuckets", selection);
-    }
+    const renames = {
```

Comment:
`renames` is probably not the best name but I really can't think of something better. `metricTypeToStoreMapping` seems too verbose for something only used once here